Userland/Libraries/LibJS/Runtime/Error.h

```diff
@@ -20,9 +20,14 @@ class Error : public Object {
     static Error* create(GlobalObject&, String const& message);
 
     explicit Error(Object& prototype);
+    Error(String message, Object& prototype);
+    virtual void initialize(GlobalObject&) override;
     virtual ~Error() override = default;
 
     void install_error_cause(Value options);
+
+private:
+    String m_message;
```

Comment:
Seems a bit wasteful to keep this around if we only use it to define a property... the previous way wasn't great but it's a bit better in that regard IMO.

Reply:
hmm, i guess so, will revert that.

Reply:
reverted